fuel-core/src/database.rs

```diff
@@ -248,3 +260,113 @@ impl InterpreterStorage for Database {
         Ok(block.headers.producer)
     }
 }
+
+#[async_trait]
+impl RelayerDb for Database {
+    /// get validator set for current eth height
+    async fn current_validator_set(&self) -> HashMap<Address, u64> {
```

Comment:
Shouldn't we return the database error if one occurs? We shouldn't swallow the error without at least logging it.